package.json

```diff
@@ -41,6 +42,7 @@
     "eslint": "^7.30.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-prettier": "^3.4.0",
+    "husky": "*",
```

Comment:
Please specify a version here

Reply:
eslint not allowing a single quotes to commit

Reply:
Please try the format script